extensions/icons8/README.md

```diff
@@ -0,0 +1,28 @@
+<div align="center">
+  <img src="extensions/icons8/assets/Icons8-Art.svg" style="width: 96px;"/>
```

Comment:
sorry, by absolute path I meant the actual link: https://raw.githubusercontent.com/raycast/extensions/1ae0136661beca0df6f2e924707e07f0d92f1800/extensions/icons8/assets/Icons8-Art.svg

Reply:
Oh I see. Thank you

Reply:
I just did it 🙂